torque-controllers/momentum-based-yoga/initTorqueBalancingYoga.m

```diff
@@ -27,7 +27,7 @@
 % and set the environmental variable YARP_ROBOT_NAME in the .bashrc file.
 
 % Simulation time in seconds
-Config.SIMULATION_TIME = inf;   
+Config.SIMULATION_TIME = 600;
```

Comment:
This is a bit surprising. Does the simulation ends after ~10 minutes? @gabrielenava

Reply:
If this is the case, it is probably something that is worth announcing to QA.

Reply:
I understood the issue here is that it is not possible to set to `inf` the simulation time, but any value (even very big) should be ok. Can we put a very high number @diegoferigo so that the simulation will never stop in practice?

Reply:
This means that if you do not press the `Stop` button from the GUI, the controller stops after 10 minutes. Our experiments / demos are never so long and this number can be modified accordingly. The previous behavior was to keep the robot controlled in torque waiting that the user stops the simulation manually.